src/core/triggers.systemd.in

```diff
@@ -23,136 +23,71 @@
 #
 # Minimum rpm version supported: 4.13.0
 
-%transfiletriggerin -P 900900 -p <lua> -- @systemunitdir@ /etc/systemd/system
--- This script will run after any package is initially installed or
--- upgraded. We care about the case where a package is initially
--- installed, because other cases are covered by the *un scriptlets,
--- so sometimes we will reload needlessly.
+%transfiletriggerin -P 900900 -- @systemunitdir@ /etc/systemd/system
+# This script will run after any package is initially installed or
+# upgraded. We care about the case where a package is initially
+# installed, because other cases are covered by the *un scriptlets,
+# so sometimes we will reload needlessly.
+if test -d /run/systemd/system; then %{_bindir}/systemctl daemon-reload; fi
```

Comment:
is there any benefit in doing this one line? why not break this the way people would expect, i.e. in three lines?

Reply:
I'm fine doing that if you prefer.  I thought the one-liner emphasized just how much simpler shell is over Lua, but again I can easily change it.

Reply:
we tend to prefer readability over density

Reply:
Ok, updated to be multi-line :arrow_down: